lib/rules/no-shadow-route-definition.js

```diff
@@ -21,7 +22,7 @@ function buildSourceLocationString(routeInfo) {
 }
 
 function buildRouteErrorInfo(routeInfo) {
-  return `"${routeInfo.name}" (${routeInfo.fullPath}, ${buildSourceLocationString(routeInfo)})`;
+  return `"${routeInfo.name}" ("${routeInfo.fullPath}", ${buildSourceLocationString(routeInfo)})`;
```

Comment:
Output is wrapped in double quote marks for empty string paths display. Example: `{ path: ""}`